src/Integration.UnitTests/Suppression/SonarQubeIssuesProviderTests.cs

```diff
@@ -83,22 +83,19 @@ public void Constructor_ThrowsOnNull()
         {
             string validProjectKey = "key1";
 
-            Action op = () => new SonarQubeIssuesProvider(null, validProjectKey, mockTimerFactory.Object, testLogger, threadHandling.Object);
+            Action op = () => new SonarQubeIssuesProvider(null, validProjectKey, testLogger);
```

Comment:
We only need to test the public constructor arguments.
The timer factory and thread handling objects are only on the internal constructor, so they don't need to be tested here.